packages/schematics/jest.config.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     '^.+\\.tsx?$': 'ts-jest'
   },
   testRegex: '\\.spec\\.tsx?$',
-  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node']
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  testPathIgnorePatterns: ['/node_modules/', '/templates/']
```

Comment:
Because I have an example Cypress spec file in the templates. Alternative would be to use template path parameters to avoid the `.spec.ts` naming pattern that Jest searches for.